statsd/statsd.go

```diff
@@ -497,9 +497,6 @@ func (c *Client) FlushTelemetryMetrics() ClientMetrics {
 }
 
 func (c *Client) send(m metric) error {
-	m.globalTags = c.Tags
```

Comment:
I think removing the global tags from telemetry metrics makes sense, but note that in the future we may want to allow the telemetry metrics to have global tags as well (this can help differentiate the telemetry metrics coming from 2 different clients with the same version running on the same host). Let's follow up in chat.